src/shape/bounding-box.js

```diff
@@ -318,11 +318,12 @@ Object.assign(pc, function () {
          * @name pc.BoundingBox#compute
          * @description Compute the size of the AABB to encapsulate all specified vertices.
          * @param {number[]|Float32Array} vertices - The vertices used to compute the new size for the AABB.
+         * @param {number} [numVerts] - Number of verticies to use from the beginning of vertices array. All vertices are used if not specified.
          */
-        compute: function (vertices) {
+        compute: function (vertices, numVerts) {
             var min = tmpVecA.set(vertices[0], vertices[1], vertices[2]);
             var max = tmpVecB.set(vertices[0], vertices[1], vertices[2]);
-            var numVerts = vertices.length / 3;
+            numVerts = numVerts || vertices.length / 3;
```

Comment:
If you pass numVerts === 0, you actually compute for all verts here, right?

Reply:
I guess that's ok? I updated the description of numVerts param to make this clear as well:

         * @param {number} [numVerts] - Number of vertices to use from the beginning of vertices array. All vertices are used if 0 or not specified.

Reply:
I dunno. I'd almost expect 0 to make it early out and not update the bbox.

Reply:
fair point, I've done that.

Reply:
If numVerts is passed in as 0, this function will calculate bounds over all verts. Is that the intention?

Btw, just realized this bounds calc will also benefit from packed positions.

Reply:
this has been changed yesterday - when 0 is passed in, code early outs.